month_end/models/month_end.py

```diff
@@ -0,0 +1,125 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2017 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+import datetime
+import calendar
+from odoo import api, models, fields
+
+
+class MonthEnd(models.Model):
```

Comment:
Since this model has no fields in database, you can use `models.AbstractModel` to avoid creating a table in the database.